owl/EASE-OBJ.owl

```diff
@@ -993,6 +990,18 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#AestheticDesign -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#AestheticDesign">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Design"/>
+        <rdfs:comment>Aesthetic design *describes* Aesthetics quality of an object.
+
+Aesthetics is the philosophical study of beauty and taste. The term stems from the Greek word “aisthetikos,” meaning “of sense perception,” and is related to the study of sensory values. In design, aesthetics refers to the visual attractiveness of a product. Visual aesthetics have these key elements: Color, Shape, Pattern, Line, Texture, Visual weight, Balance, Scale, Proximity and Movement.</rdfs:comment>
```

Comment:
you use strange double quotes here. Why not using the regular ones?
Also I think the comma should not be quoted.